### PR TITLE
BZ#1881453: Document unused namespace for Manila CSI driver operator after upgrading from 4.5 to 4.6

### PR DESCRIPTION
**Pull request by jldohmann**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1881453

OCP Version: 4.6

Current 4.6 doc: https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-csi-driver-cso

Direct doc preview link: https://deploy-preview-37066--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-6-csi-driver-cso

### CODE REVIEW

**Review by tombarron — APPROVED** [review]
This looks right. I wasn't directly involved with these changes but remember Mike Fedosin making adjustments in namespace to conform with more general openshift conventions.

**Review by bergerhoffer — COMMENTED** [review]
Just a few suggestions!

File: release_notes/ocp-4-6-release-notes.adoc
Comment (jldohmann) [reply release_notes/ocp-4-6-release-notes.adoc]: All makes sense, thank you!

**Review by bergerhoffer — COMMENTED** [review]
This LGTM now thanks for the updates! Just ping me once you have QE approval and are ready for me to merge!

File: release_notes/ocp-4-6-release-notes.adoc
Comment (duanwei33) [inline release_notes/ocp-4-6-release-notes.adoc:759]: Hi, I am thinking if using the **need** instead of **must** for the namespace deletion.

Comment (jldohmann) [reply release_notes/ocp-4-6-release-notes.adoc:759]: I think it's best to leave it as is: we follow the IBM style guide, which recommends against using phrasal verbs like "need to" e.g. "need to be deleted by a cluster administrator"

Comment (duanwei33) [reply release_notes/ocp-4-6-release-notes.adoc:759]: @jldohmann Hi, here I mean deleting the `openshift-manila-csi-driver-operator` namespace is not mandatory, it depends on the customer who installed the operator in 4.5, like if there is something others in this namespace. So I suggest to not use the "must", okay for me to use other words to match the IBM style guide.

Comment (jldohmann) [reply release_notes/ocp-4-6-release-notes.adoc:759]: I see, thanks for clarifying. I will update the sentence to communicate this.

**Comment by jldohmann** [comment]
cf33e3a...0033514 must --> can

**Pull request merged**
